src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -0,0 +1,336 @@
+      Left: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThanOrEqual) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+          Left: ILocalReferenceExpression: m (OperationKind.LocalReferenceExpression, Type: System.Int32)
+          Right: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+      Right: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThanOrEqual) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+          Left: ILocalReferenceExpression: m (OperationKind.LocalReferenceExpression, Type: System.Int32)
+          Right: ILocalReferenceExpression: p (OperationKind.LocalReferenceExpression, Type: System.Int32)
+  IBlockStatement (1 statements) (OperationKind.BlockStatement)
+    IExpressionStatement (OperationKind.ExpressionStatement)
+      IInvocationExpression (static void System.Console.WriteLine(System.String value)) (OperationKind.InvocationExpression, Type: System.Void)
+        IArgument (Matching Parameter: value) (OperationKind.Argument)
+          ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: Nothing is larger than m.)
+";
+            VerifyOperationTreeForTest<IfStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17601, "https://github.com/dotnet/roslyn/issues/17601")]
+        public void IIfstatementWithElseIfCondition()
+        {
+            string source = @"
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+class P
+{
+    private void M()
+    {
+        int m = 9;
+        int n = 7;
+        /*<bind>*/if (n > 20)
+        {
+            Console.WriteLine(""Result1"");
+        }
+        else if(m > 10)
+        {
+            Console.WriteLine(""Result2"");
+        }
+        else
+        {
+            Console.WriteLine(""Result3"");
+        }/*</bind>*/
+
+    }
+}
+";
+            string expectedOperationTree = @"
+IIfStatement (OperationKind.IfStatement)
+  Condition: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+      Left: ILocalReferenceExpression: n (OperationKind.LocalReferenceExpression, Type: System.Int32)
+      Right: ILiteralExpression (Text: 20) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 20)
+  IBlockStatement (1 statements) (OperationKind.BlockStatement)
+    IExpressionStatement (OperationKind.ExpressionStatement)
+      IInvocationExpression (static void System.Console.WriteLine(System.String value)) (OperationKind.InvocationExpression, Type: System.Void)
+        IArgument (Matching Parameter: value) (OperationKind.Argument)
+          ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: Result1)
+  IIfStatement (OperationKind.IfStatement)
```

Comment:
Seems to be the case where `if... elseif.. else` has been converted into `if... else (if... else)` due to lowering.